package.json

```diff
@@ -192,7 +192,7 @@
     "rison-node": "1.0.0",
     "rxjs": "5.4.3",
     "script-loader": "0.7.2",
-    "semver": "5.1.0",
+    "semver": "5.5.0",
```

Comment:
Can we update this to use a caret and track the most recent minor?

Reply:
Yes ofc @tylersmalley! I have already done it.